### PR TITLE
riscv/esp32c3: Initialize the BLE Mac

### PR DESCRIPTION
**Pull request by xiewenxiang**
## Summary

Initialize the BLE static Address when initializing the Bluetooth Controller.

### CODE REVIEW

**Comment by acassis** [comment]
Hi @xiewenxiang it is returning an error:
```
riscv64-unknown-elf-ld: /github/workspace/sources/nuttx/staging/libarch.a(esp32c3_ble_adapter.o): in function `read_mac_wrapper':
esp32c3_ble_adapter.c:(.iram1+0x98): undefined reference to `esp_read_mac'
```
In IDF it is defined at components/esp_common/src/mac_addr.c

**Comment by xiaoxiang781216** [comment]
@xiewenxiang please fix this warning:
```
/home/runner/work/incubator-nuttx/incubator-nuttx/nuttx/arch/risc-v/src/esp32c3/esp32c3_wireless.c:353:7: error: Right brace must be followed by a blank line
```

**Comment by xiewenxiang** [comment]
> Hi @xiewenxiang it is returning an error:
> 
> ```
> riscv64-unknown-elf-ld: /github/workspace/sources/nuttx/staging/libarch.a(esp32c3_ble_adapter.o): in function `read_mac_wrapper':
> esp32c3_ble_adapter.c:(.iram1+0x98): undefined reference to `esp_read_mac'
> ```
> 
> In IDF it is defined at components/esp_common/src/mac_addr.c

I have modified it.

**Comment by xiewenxiang** [comment]
> @xiewenxiang please fix this warning:
> 
> ```
> /home/runner/work/incubator-nuttx/incubator-nuttx/nuttx/arch/risc-v/src/esp32c3/esp32c3_wireless.c:353:7: error: Right brace must be followed by a blank line
> ```

@xiaoxiang781216  OK, I have fixed it.

File: arch/risc-v/src/esp32c3/esp32c3_wireless.c [hunk 1]
```diff
@@ -29,9 +29,17 @@
...
+#include "esp32c3_wireless.h"
 #include "esp32c3.h"
 #include "espidf_wifi.h"
 
+/****************************************************************************
+ * Pre-processor Definitions
+ ****************************************************************************/
+
+#define MAC_ADDR0_REG (DR_REG_EFUSE_BASE + 0x044)
+#define MAC_ADDR1_REG (DR_REG_EFUSE_BASE + 0x048)
```
Comment (gustavonihei) [inline arch/risc-v/src/esp32c3/esp32c3_wireless.c:41]:
```suggestion
#define MAC_ADDR0_REG EFUSE_RD_MAC_SPI_SYS_0_REG
#define MAC_ADDR1_REG EFUSE_RD_MAC_SPI_SYS_1_REG
```
It is best to reuse the definitions from https://github.com/apache/incubator-nuttx/blob/master/arch/risc-v/src/esp32c3/hardware/esp32c3_efuse.h#L815

**Comment by xiewenxiang** [comment]
Hi @xiaoxiang781216 @gustavonihei 
Could you please merge this PR..

File: arch/risc-v/src/esp32c3/esp32c3_wireless.c (hunk 1 above)
Comment (xiewenxiang) [reply arch/risc-v/src/esp32c3/esp32c3_wireless.c]: done

**Review by gustavonihei — APPROVED** [review]

**Pull request merged**